src/ViewController.js

```diff
@@ -107,8 +110,11 @@ class ViewController extends HasModels {
       .sort(pageOptions.sortField + ' ' + pageOptions.sortDirection)
       .limit(pageOptions.itemsPerPage)
       .skip((pageOptions.currentPage-1)*pageOptions.itemsPerPage)
-    if (this.populate) {
-      find.populate(this.populate)
+    if (this.populate.length) {
```

Comment:
My bias is to not wrap the for-loop in an if-clause in cases like this – the for-loop checks for the empty case anyway. Same thing for line 127.

Also surprised you didn't use the new spread syntax (`find.populate(...p)`). ;-)